src/Specs.as

```diff
@@ -355,33 +356,32 @@ public class Specs {
 		["-"],
 		["item %d.listItem of %m.list",						"r", 12, "getLine:ofList:"],
 		["length of %m.list",								"r", 12, "lineCountOfList:"],
-		["%m.list contains %s?",								"b", 12, "list:contains:"],
+		["%m.list contains %s?",							"b", 12, "list:contains:"],
 		["-"],
 		["show list %m.list",								" ", 12, "showList:"],
 		["hide list %m.list",								" ", 12, "hideList:"],
 
 		// obsolete blocks from Scratch 1.4 that may be used in older projects
-		["play drum %n for %n beats",			" ", 98, "drum:duration:elapsed:from:", 1, 0.25], // Scratch 1.4 MIDI drum
-		["set instrument to %n",				" ", 98, "midiInstrument:", 1],
-		["loud?",								"b", 98, "isLoud"],
+		["play drum %n for %n beats",			" ", 98, "drum:duration:elapsed:from:", 48, 0.25], // Scratch 1.4 MIDI drum
+		["set instrument to %n",				" ", 98, "midiInstrument:", 1], // Scratch 1.4 MIDI instrument
+		["loud?",								"b", 98, "isLoud"], // Loudness > 10 (30 in 1.4)
 
 		// obsolete blocks from Scratch 1.4 that are converted to new forms (so should never appear):
 		["abs %n",								"r", 98, "abs"],
 		["sqrt %n",								"r", 98, "sqrt"],
 		["stop script",							"f", 98, "doReturn"],
 		["stop all",							"f", 98, "stopAll"],
-		["switch to background %m.costume",		" ", 98, "showBackground:", "backdrop1"],
-		["next background",						" ", 98, "nextBackground"],
+		["switch backdrop to %m.costume",		" ", 98, "showBackground:", "backdrop1"],
+		["next backdrop",						" ", 98, "nextBackground"],
```

Comment:
These block specs need to stay the way they were: they need to match the specs from Scratch 1.4 in order to import old projects correctly.